test/unit-tests/function/matrix/eigs.test.js

```diff
@@ -16,23 +16,36 @@ describe('eigs', function () {
   it('follows aiao-mimo', function () {
     const realSymArray = eigs([[1, 0], [0, 1]])
     assert(Array.isArray(realSymArray.values) && typeof realSymArray.values[0] === 'number')
-    assert(Array.isArray(realSymArray.vectors) && typeof realSymArray.vectors[0][0] === 'number')
+    for (let ix = 0; ix < 2; ++ix) {
+      assert(Array.isArray(realSymArray.eigenvectors[ix].vector))
+    }
+    assert(typeof realSymArray.eigenvectors[0].vector[0] === 'number')
 
     const genericArray = eigs([[0, 1], [-1, 0]])
     assert(Array.isArray(genericArray.values) && genericArray.values[0] instanceof Complex)
-    assert(Array.isArray(genericArray.vectors) && genericArray.vectors[0][0] instanceof Complex)
+    for (let ix = 0; ix < 2; ++ix) {
+      assert(Array.isArray(genericArray.eigenvectors[ix].vector))
+    }
+    assert(genericArray.eigenvectors[0].vector[0] instanceof Complex)
 
     const realSymMatrix = eigs(matrix([[1, 0], [0, 1]]))
     assert(realSymMatrix.values instanceof Matrix)
     assert.deepStrictEqual(size(realSymMatrix.values), matrix([2]))
-    assert(realSymMatrix.vectors instanceof Matrix)
-    assert.deepStrictEqual(size(realSymMatrix.vectors), matrix([2, 2]))
+    for (let ix = 0; ix < 2; ++ix) {
```

Comment:
Would it be worth to create a little helper function to assert the output of `eigs`? The for loop is repeated quite a couple of times.

Reply:
Again, I think I've done the refactor you were hoping, but happy for advice on better versions...

Reply:
Thanks! Replacing the `for` loop with a `forEach` already makes it more compact and easy to follow, the helper function is a one-liner now.